src/Microsoft.Azure.Batch.SoftwareEntitlement.Server/readme.md

```diff
@@ -77,3 +79,17 @@ The service will return HTTP status 400 and the response body will be empty if:
 * The software entitlement token is missing, invalid, or corrupt;
 * The request is badly formed; or
 * The `api-version` specified on the URL is invalid.
+
```

Comment:
Shouldn't we add something like "The software entitlement token has expired." to the relevant error cases?

Reply:
Added details to the *400* response code, just above this change.